releng/config-forker/main.go

```diff
@@ -39,6 +39,7 @@ const (
 	periodicIntervalAnnotation   = "fork-per-release-periodic-interval"
 	cronAnnotation               = "fork-per-release-cron"
 	replacementAnnotation        = "fork-per-release-replacements"
+	deletionAnnotation           = "fork-per-release-deletions"
```

Comment:
nitpick: what would you think about calling this something like `fork-per-release-remove-labels`? It took me a moment to determine exactly what this was doing (could just be me :sweat_smile:). We could also consider building this in with `fork-per-release-replacements` by defining a new operator other than `->` (maybe `X`), but I tend to like what you have added here as a simpler solution :+1:

Reply:
>what would you think about calling this something like fork-per-release-remove-labels

I imagine one would like to use `fork-per-release-deletions` in some scenarios other than just labels in the future. Currently the need is just for the labels but somebody might need this for other job fields as well (`fork-per-release-replacements` already performs substitutions on the container args and tags) so limiting it to just the labels might result in some maintenance problems in the future. OTOH I do not have a strong opinion here :).

>We could also consider building this in with fork-per-release-replacements by defining a new operator other than -> (maybe X)

Thought about it as well, but that was a bit more complicated and went for the simpler solution.

Thanks for the prompt peek into that!